_drafts/asking-for-help.md

```diff
@@ -0,0 +1,40 @@
+
+First, I want to note that a fear of asking for help can be compounded by very real oppressive structures and lived experience. Women, gender-diverse folks, and people of color experience vitriol online at higher rates. Further, these groups often encounter increased scrutiny at work that can make regularly asking for help more intimidating and possibly detrimental to career progress. It's important to cultivate a work environment that actively addresses power structures to create a space that encourages open, vulnerable communication and where asking for help feels comfortable to all.
+
+1. **Do some research before posting/asking**
+
+- The first step to encountering a problem is trying to solve it or at least understand it as much as you can on your own. Toss your error code on google, peruse help forums, try a few solutions. 
+
+- *Best case scenario:* Someone has already asked your exact question and the solution provided works for you! It's good practice to double-check that your question has not already been answered in an internet forum before posting.
+
+- *Worst case scenario:* No working results turn up. While this can be a frustrating process it will help you further diagnose your problem (if only by determining what doesn't work) and enable you to describe the situation more clearly. 
+
+2. **Clearly state your problem**
+
+- The quality of the answers you receive is directly related to the quality of the question posed. A good question is concise and systematically addresses the problem. Make sure that before posting any code that you properly introduce and give context to the question you are asking. Enough detail should be provided that someone could answer you without requesting more information.
+
+3. **Make sure your problem is reproducible**
+
+- It is best practice to provide a minimal reproducible example of the code that you need help with. An ideal example uses the minimal amount of code required to reproduce the problem and includes all the parts necessary to reproduce the problem including inputs and desired output. The test code should reliably reproduce the problem you're trying to solve. 
+
+- Do not use images of code. Copy and paste text and then format it as code. It helps with readability and makes it easier to run your test code.
+
+- Make sure that the code you show doesn't contain any sensitive information like personal information, passcodes, etc.
+
+4. **Be nice!**
+
+- Generally, the people you're going to for help are unpaid and volunteering their time and expertise. Kindness and understanding go a long way. 
```

Comment:
We do have a lot of folks here at the Hutch (like us and SciComp) for whom assisting with such questions is part of our jobs. Can we tweak the language to include these folks being strapped for time and possibly handling many such requests?